packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -46,16 +46,100 @@ export const ContextBundle = createContext<LiveblocksContextBundle<
  * This is an internal API, use "createLiveblocksContext" instead.
  */
 export function useLiveblocksContextBundle() {
-  const bundle = useContext(ContextBundle);
-  if (bundle === null) {
-    throw new Error("LiveblocksProvider is missing from the React tree.");
-  }
-  return bundle;
+  return (
+    useContext(ContextBundle) ??
+    raise("LiveblocksProvider is missing from the React tree.")
+  );
 }
 
 export const POLLING_INTERVAL = 60 * 1000; // 1 minute
 export const INBOX_NOTIFICATIONS_QUERY = "INBOX_NOTIFICATIONS";
 
+// --- Selector helpers ------------------------------------------------- {{{
```

Comment:
All of these helpers haven't been changed. They're just moved out of the closure and into the top-level, which will make it easier to reuse them in the top-level hooks soon.